subiquitycore/file_util.py

```diff
@@ -30,6 +30,8 @@
 
 
 def set_log_perms(target, *, group_write=False, mode=None, group=_DEF_GROUP):
+    if os.getenv("SNAP_CONFINEMENT", "classic") == "strict":
```

Comment:
@dbungert to be precise, the problem is with chown() further down. The seccomp template for strict snaps prevents a process from executing chown() with target user/group other than -1 or root. In this case https://github.com/canonical/subiquity/blob/53180bbc355f91e569427b97d73ac6d4825e1407/subiquitycore/log.py#L26 is being called which attempts to chown with group `adm`. I can push a patch to skip the chown call when executing under a strict confinement. Though I think it may be preferrable to update `log.setup_logger()` to use `root` in such scenario, wdyt?

Reply:
I've opened https://github.com/canonical/subiquity/pull/1903 please have a look